exonum-java-binding/cryptocurrency-demo/start-cryptocurrency-demo.sh

```diff
@@ -13,43 +13,37 @@ set -eu -o pipefail
 
 header "DETECTING ENVIRONMENT"
 
-# EXONUM_HOME shoudl be set to the root directory of Exonum Java App installed
-if [ -z "${EXONUM_HOME+x}" ];
+# Check whether the Exonum Java App is installed.
+EXONUM_JAVA_APP="exonum-java"
+command -v ${EXONUM_JAVA_APP} >/dev/null 2>&1 || { echo >&2 "Please install the Exonum Java App and make sure that 'exonum-java' binary is available via PATH. Aborting."; exit 1; }
+
+# Check that path to the JVM library is set.
+if [ -z "${JAVA_LIB_DIR+x}" ];
 then
-    echo "ERROR: EXONUM_HOME variable is empty. Please point the EXONUM_HOME to the root of installed Exonum Java App."
+    echo "ERROR: JAVA_LIB_DIR variable is not set. Please make sure you executed 'source ../tests_profile'"
     exit 1
 fi
-echo "EXONUM_HOME=${EXONUM_HOME}"
 
-EXONUM_JAVA_APP="${EXONUM_HOME}/exonum-java"
+export LD_LIBRARY_PATH="${JAVA_LIB_DIR}"
+echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
 
 # Find the latest version of artifact and build if not found.
 ARTIFACT_PATH="$(find ./target -type f -name exonum-java-binding-cryptocurrency-demo-*-artifact.jar)"
 
-if [ -z "${ARTIFACT_PATH}+x" ];
+if [ -z "${ARTIFACT_PATH}" ];
 then
-    mvn install
+    mvn package
```

Comment:
Have you tested that with an installed application **and** a clean repository? It can't work for there won't be a native library required for tests.

Reply:
I did for updated instructions steps after `mvn clean`.

Reply:
`mvn clean` does not clean the native target